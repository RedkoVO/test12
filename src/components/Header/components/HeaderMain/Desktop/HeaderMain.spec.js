import * as React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import HeaderMain from './'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter keyLength={0}>
        <HeaderMain handleLogout={() => {}} balance={{}} />
      </MemoryRouter>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
