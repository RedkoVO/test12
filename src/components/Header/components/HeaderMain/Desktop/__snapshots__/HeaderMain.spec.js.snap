// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="HeaderMain-root-1"
>
  <ul
    className="MenuNavigation-root-2"
  >
    <li
      className={undefined}
    >
      <div
        className="MenuNavigation-logout-11"
        onClick={[Function]}
      >
        Logout
      </div>
    </li>
    <li>
      <a
        className="MenuNavigation-headerMenuItem-3"
        href="/"
        onClick={[Function]}
      >
        <i
          className="MenuNavigation-menuIcon-4 MenuNavigation-dashboard-5"
        />
        <br />
         Dashboard
      </a>
    </li>
    <li>
      <a
        className="MenuNavigation-headerMenuItem-3"
        href="/"
        onClick={[Function]}
      >
        <i
          className="MenuNavigation-menuIcon-4 MenuNavigation-walletIcon-6"
        />
        <br />
         Wallet
      </a>
    </li>
    <li>
      <a
        className="MenuNavigation-headerMenuItem-3"
        href="/"
        onClick={[Function]}
      >
        <i
          className="MenuNavigation-menuIcon-4 MenuNavigation-shop-7"
        />
        <br />
         Shop
      </a>
    </li>
    <li>
      <a
        className="MenuNavigation-headerMenuItem-3"
        href="/games"
        onClick={[Function]}
      >
        <i
          className="MenuNavigation-menuIcon-4 MenuNavigation-games-8"
        />
        <br />
         Games
      </a>
    </li>
    <li>
      <a
        className="MenuNavigation-headerMenuItem-3"
        href="/"
        onClick={[Function]}
      >
        <i
          className="MenuNavigation-menuIcon-4 MenuNavigation-friends-9"
        />
        <br />
         Friends
      </a>
    </li>
    <li>
      <a
        className="MenuNavigation-headerMenuItem-3"
        href="/"
        onClick={[Function]}
      >
        <i
          className="MenuNavigation-menuIcon-4 MenuNavigation-settings-10"
        />
        <br />
         Settings
      </a>
    </li>
  </ul>
  <div
    className="UserBlock-root-12"
  >
    <div
      className="UserBlock-demoTitle-20"
    >
      DEMO VERSION
    </div>
    <div
      className="UserBlock-user-13"
    >
      <div
        className="UserBlock-userBlock-14"
      >
        <p
          className="UserBlock-userHello-15"
        >
           Hello, Ben A 
          <br />
          <span
            className="UserBlock-userVerification-16"
          >
            KYC VERIFICATED
          </span>
        </p>
        <div
          className="UserBlock-userPhoto-17"
        >
          <img
            alt=""
            src="user.png"
          />
        </div>
        <p
          className="UserBlock-userBallance-18"
        >
           dcb
          <br />
          <span
            className="UserBlock-userBallDescr-19"
          >
            YOUR BALANCE
          </span>
        </p>
      </div>
    </div>
  </div>
</header>
`;
